components/Button/TabButton.tsx

```diff
@@ -19,6 +19,7 @@ export type TabButtonProps = {
 	nofill?: boolean;
 	isRounded?: boolean;
 	onClick?: () => any;
+	flat?: boolean;
```

Comment:
Annoying we still have this prop style around but let's deal with it in a different component focused PR